tests/tests/test_filetransfer_cli.py

```diff
@@ -18,18 +18,19 @@
 
 from tempfile import NamedTemporaryFile
 
-from ..common_setup import standard_setup_one_client_bootstrapped
-from ..MenderAPI import authentication, devauth, get_container_manager, logger
-from .common_connect import wait_for_connect
+from ..common_setup import standard_setup_one_client_bootstrapped, enterprise_no_client
+from ..MenderAPI import authentication, get_container_manager, logger, DeviceAuthV2
+from .common_connect import prepare_env_for_connect, wait_for_connect
 from .common import md5sum
 from .mendertesting import MenderTesting
 
 
-class TestFileTransferCLI(MenderTesting):
+class TestFileTransferCLIBase(MenderTesting):
```

Comment:
Just so that pytest does not pick this up as a test class (without tests) I would add an underscore prefix to the class name.
```suggestion
class _TestFileTransferCLIBase(MenderTesting):
```